test/libsolidity/ASTLegacyJSON.cpp

```diff
@@ -266,15 +266,15 @@ BOOST_AUTO_TEST_CASE(function_type)
 	BOOST_CHECK_EQUAL(argument["attributes"]["name"], "x");
 	BOOST_CHECK_EQUAL(argument["attributes"]["type"], "function () payable external returns (uint256)");
 	Json::Value funType = argument["children"][0];
-	BOOST_CHECK_EQUAL(funType["attributes"]["constant"], false);
+	BOOST_CHECK_EQUAL(funType["attributes"]["stateMutability"], "payable");
 	BOOST_CHECK_EQUAL(funType["attributes"]["payable"], true);
```

Comment:
@chriseth @ekpyron Hm, do we also want to remove this and prefer `stateMutability`? Not as part of this PR though.

Reply:
Yes, I think it makes sense to remove ``payable`` as well and to always use ``stateMutability`` to avoid information duplication.